custom-connector-example/src/main/java/com/amazonaws/appflow/custom/connector/example/handler/SalesforceConfigurationHandler.java

```diff
@@ -92,4 +99,16 @@ public DescribeConnectorConfigurationResponse describeConnectorConfiguration(fin
                 .supportedWriteOperations(SalesforceConnectorConfiguration.supportedWriteOperations())
                 .build();
     }
-}
+
+    private String buildSalesforceUserInfoRequest(final ValidateCredentialsRequest request) {
```

Comment:
I think we are expecting this as connector runtime setting so we need to add this as connctor runtime setting in configuration as well.